File: library/src/main/java/com/getstream/sdk/chat/style/ChatFonts.java
package com.getstream.sdk.chat.style;

import android.graphics.Typeface;
import android.widget.TextView;

import com.getstream.sdk.chat.utils.roundedImageView.CircularImageView;

public interface ChatFonts {
    void setFont(TextStyle fontStyle, TextView textView);
    void setFont(TextStyle fontStyle, CircularImageView imageView, float factor);
    Typeface getFont(TextStyle textStyle);
}

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/style/ChatFonts.kt
package io.getstream.chat.android.ui.common.style

import android.graphics.Typeface
import android.widget.TextView

public interface ChatFonts {
    public fun setFont(textStyle: TextStyle, textView: TextView)
    public fun setFont(textStyle: TextStyle, textView: TextView, defaultTypeface: Typeface = Typeface.DEFAULT)
    public fun getFont(textStyle: TextStyle): Typeface?
}
